packages/styles/src/tile.scss

```diff
@@ -831,25 +845,44 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
       }
 
       .#{$block}__action-close {
-        @include fd-set-height(1.625rem);
-        @include reset-position-absolute();
-        @include fd-flex-center();
-
+        @include fd-set-height(1.375rem);
+        @include fd-set-width(1.375rem);
+        position: relative;
         padding-block: 0;
         padding-inline: 0.25rem;
+        font-size: 0.75rem;
+        border-radius: 50%;
+        background: var(--sapButton_Background);
+        border: var(--sapButton_BorderColor);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+        top: -0.625rem;
+        right: -0.325rem;
```

Comment:
when you use absolute positioning and set either the right or the left value, you need to handle RTL (right-to-left support)
We have a mixin that handles this:
```
@mixin fd-set-position-right($right) {
  right: $right;

  @include fd-rtl() {
    right: auto;
    left: $right;
  }
}
```

Reply:
done!